src/components/LottieControl.jsx

```diff
@@ -33,6 +49,7 @@ export default class LottieControl extends Component {
       <button style={buttonStyle} onClick={() => this.setState({isStopped: true})}>stop</button>
       <button style={buttonStyle} onClick={() => this.setState({isStopped: false})}>play</button>
       <button style={buttonStyle} onClick={() => this.setState({isPaused: !this.state.isPaused})}>pause</button>
+      <button style={buttonStyle} onClick={() => this.handleNextAnimation()}>next animation</button>
```

Comment:
I know this is a pretty rough draft of things but we should add 'make a button component' as an item to our todo list. (If we don't have one already)